spartan/aztec-network/templates/metrics.yaml

```diff
@@ -17,6 +16,8 @@ spec:
         {{- include "aztec-network.selectorLabels" . | nindent 8 }}
         app: metrics
     spec:
+      securityContext:
+        fsGroup: 472 # required for Grafana write access to storage volume
```

Comment:
472 could be in the values file so it isn't a raw magic number

Reply:
Done. Moved to values file.